pkg/metrics/metrics.jsx

```diff
@@ -387,16 +390,33 @@ class CurrentMetrics extends React.Component {
                     const is_user = k.match(/^user.*user@\d+\.service.+/);
                     const label = k.replace(/.*\//, '').replace(/\.service$/, '');
                     // only keep cgroup basenames, and drop redundant .service suffix
-                    merged.push([label, v, is_user]);
+                    merged.push([label, v, is_user, false]);
+                }
+                // filter out podman containers
+                const matches = k.match(podmanSystemRe);
+                if (matches && v) {
+                    // truncate to 12 chars like the podman output
+                    const containerid = matches.groups.containerid.substr(0, 12);
+                    const is_user = k.match(/^user.*user@\d+\.service.+/);
+                    merged.push([containerid, v, is_user, true]);
                 }
             });
             merged.sort((a, b) => b[1] - a[1]);
             return merged.slice(0, n);
         }
 
-        function serviceRow(name, value, is_user) {
+        function serviceClickHandler(name, is_user, is_container) {
```

Comment:
Let's rename "service" to "cgroup" then, as these are now not just systemd services any more.

Reply:
Ok